migrator/migrations/n_30_to_n_31_postgres_network_flows/migration_test.go

```diff
@@ -86,6 +86,14 @@ func (s *postgresMigrationSuite) verify(flowStore store.FlowStore, flows []*stor
 		return flows[i].LastSeenTimestamp.Compare(flows[j].LastSeenTimestamp) < 0
 	})
 	for i, flow := range flows {
+		// Postgres Datetime columns only have microsecond granularity for timestamps.
+		// Adapt the input data to take this into account.
+		flowLastSeenTimestampNanos := flow.GetLastSeenTimestamp().GetNanos()
+		flowLastSeenTimestampNanos /= 1000
+		flowLastSeenTimestampNanos *= 1000
```

Comment:
How about extracting this to a function. It take a while to understand what's going on here